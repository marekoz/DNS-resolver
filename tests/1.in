-r -s 8.8.8.8 example.com