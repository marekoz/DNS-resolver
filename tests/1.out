Authoritative: No, Recursive: Yes, Truncated: No
Question section (1)
  example.com., Type: A, Class: IN
Answer section (1)
  example.com., Type: A, Class: IN, TTL: <number>, Data length: 4, 93.184.216.34
Authority section (0)
Additional section (0)
